tests/Fixer/PhpUnit/PhpUnitSetUpTearDownVisibilityFixerTest.php

```diff
@@ -268,6 +268,24 @@ public function setUp() {}
 
     public function tearDown() {}
 }
+',
+        ];
+
+        yield 'It does not touch anonymous class' => [
```

Comment:
I would prefer a testcase, in which we show that test-level method was fixed, while anon-class level method was ignored
example https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8365/files#diff-5575a51f30af7ecefd94467d7bf15488ac7ca728ebb6c7846ea94b44afd89a22R278-R281